# Message storing the points of users.
# Header header
uint8[] users
geometry_msgs/PointStamped[] people_points
bounding_box[] boxes